### PR TITLE
Home

### PR DESCRIPTION
**Pull request by uday-nv-work**

### CODE REVIEW

File: css/component.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	--COLOR-SECONDARY: #27e4b5;
+	--COLOR-SOCIAL-ICON: #308ea5;
+	--PRIMARY-HOVER: #7a9ea7;
+	--NOTIFICATION-COLOR: #7cf916;
+	--offgreen: #bacfc0;
+	--darkerofseconadary: #2195d8;
+	--BOX-SHADOW: rgb(175, 152, 152);
+	--BOX-SHADOW-LIGHTER: rgb(78, 76, 37);
+	--ALERT-PRIMARY-COLOR: #2563eb;
+	--ALERT-SUCCESS-COLOR: rgb(13, 224, 31);
```
Comment (rutu14) [inline css/component.css]: Suggestion - Try to use one code value for color.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+.card-container:hover {
+	transform: scale(1.02) rotate(2deg);
+}
+.input {
+	padding: 0 16px;
+	border-radius: 2px 0 0 2px;
+	border: 0;
+	outline: 0 none;
+	font-size: 14px;
+	height: 36px;
```
Comment (rutu14) [inline css/style.css]: Unit Change - The height, padding and font value's unit can be changed to rems.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	margin: auto;
+	display: inline;
+	outline: 0;
+	border-radius: 5px;
+}
+.rt {
+	top: 120%;
+	right: 0;
+	z-index: 2;
+	color: aliceblue;
```
Comment (rutu14) [inline css/style.css]: Color Variable - Assign a color variable.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+	vertical-align: middle;
+}
+.lt {
+	top: 120%;
+	left: 0;
+	z-index: 2;
+	color: aliceblue;
```
Comment (rutu14) [inline css/style.css:67]: Color Variable - Assign a color variable.

**Review by rutu14 — CHANGES_REQUESTED** [review]
UI Suggestion: 

- The navbar buttons of Cart and Wishlist can have both text and icon, or only icon.
- The container below the navbar doesn't have spacing between the rows.
- The ADD TO CART button can have different color for font text. The current white is not readable over blue.

UI Appreciation: The dark mode and shadows looks amazing.

**Comment by Ranjeetk489** [comment]
The logo isn't loading up.

File: css/component.css (hunk 1 above)
Comment (Ranjeetk489) [reply css/component.css]: +1 this change is pending

**Comment by Ranjeetk489** [comment]
the uploaded avatars can be compressed using online tool...it'll decrease site loading time

**Review by Ranjeetk489 — APPROVED** [review]

**Review by rutu14 — APPROVED** [review]

**Pull request merged**